mne_bids/read.py

```diff
@@ -166,3 +157,73 @@ def read_raw_bids(bids_fname, bids_root, verbose=True):
         raw.info['bads'] = list(unique_bads)
 
     return raw
+
+
+def get_head_mri_trans(bids_fname, bids_root):
+    """Produce transformation matrix from MEG and MRI landmark points.
+
+    Will attempt to read the landmarks of Nasion, LPA, and RPA from the sidecar
+    files of (i) the MEG and (ii) the T1 weighted MRI data. The two sets of
+    points will then be used to calculate a transformation matrix from HEAD
+    coordinates to MRI coordinates.
+
+    Parameters
+    ----------
+    bids_fname : str
+        Full name of the MEG data file (not a path)
+    bids_root : str
+        Path to root of the BIDS folder
+
+    Returns
+    -------
+    trans : instance of mne.transforms.Transform
+        The data transformation matrix from HEAD to MRI coordinates
+
+    """
+    try:  # pragma: no cover
+        import nibabel as nib
+    except ImportError:  # pragma: no cover
+        raise ImportError('This function requires nibabel.')
+
+    # Get the sidecar file for MRI landmarks
+    t1w_json_path = _find_matching_sidecar(bids_fname, bids_root, 'T1w.json')
+
+    # Get MRI landmarks from the JSON sidecar
+    with open(t1w_json_path, 'r') as f:
+        t1w_json = json.load(f)
+    mri_coords_dict = t1w_json['AnatomicalLandmarkCoordinates']
```

Comment:
This could fail, should we raise an error if the key doesn't exist?

Reply:
good point, thanks. I made this more robust: https://github.com/mne-tools/mne-bids/pull/211/commits/37898f9a9c1f29a176443c31b8069a32a2a58c8a